Source/SwiftLintFramework/Rules/LineLengthRule.swift

```diff
@@ -87,4 +92,14 @@ public struct LineLengthRule: ConfigurationProviderRule, SourceKitFreeRule {
         return modifiedString
     }
 
+    private func stripUrls(fromSourceString sourceString: String) -> String {
+        let types = NSTextCheckingResult.CheckingType.link.rawValue
+        let range = NSRange(location: 0, length: sourceString.characters.count)
```

Comment:
this should be `sourceString.bridge().length` since the number of characters does not always equal the NSString length value.